apps/internal/oauth/ops/authority/authority.go

```diff
@@ -110,16 +110,20 @@ const (
 
 // AuthParams represents the parameters used for authorization for token acquisition.
 type AuthParams struct {
-	AuthorityInfo     Info
-	CorrelationID     string
-	Endpoints         Endpoints
-	ClientID          string
-	Redirecturi       string
-	HomeaccountID     string
-	Username          string
-	Password          string
-	Scopes            []string
-	AuthorizationType AuthorizeType
+	AuthorityInfo       Info
+	CorrelationID       string
+	Endpoints           Endpoints
+	ClientID            string
+	Redirecturi         string
+	HomeaccountID       string
+	Username            string
+	Password            string
+	Scopes              []string
+	AuthorizationType   AuthorizeType
+	State               string
```

Comment:
Sees you added a few.  For new ones we add, we should document them.  If you know the others enough to document them, that would be great too.